stack.yaml

```diff
@@ -11,15 +11,14 @@ packages:
 extra-deps:
   - monad-gen-0.3.0.1
   - functor-infix-0.0.5
-  - idris-1.3.1
+  - idris-1.3.2
   - cheapskate-0.1.1.1
-  - megaparsec-6.5.0
-  - neat-interpolation-0.3.2.2
+  - megaparsec-7.0.5
   - set-extra-1.4.1
   - llvm-hs-pretty-0.6.1.0
 
-  - github: grin-tech/grin
-    commit: ed7bacf715af709ed15e0f285a7fdc6d0263c8ba
+  - github: luc-tielen/grin
+    commit: a730ec167cc70487b5027d62ec80b708570ad4f8
```

Comment:
This needs to point to grin-tech/grin again after https://github.com/grin-compiler/grin/pull/56 is merged in